src/cmd/uds.go

```diff
@@ -18,6 +22,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
+//go:embed bin/uds-runtime-*
```

Comment:
Can we remove the asterisk and refer to the specific binary?

Reply:
yup, will update so we only bundle necessary binary at build time